addons/common/XEH_preInit.sqf

```diff
@@ -78,3 +79,19 @@ activateAddons GVAR(addons);
 ["CAManBase", "InitPost", CBA_fnc_randomizeFacewear] call CBA_fnc_addClassEventHandler;
 
 ADDON = true;
```

Comment:
All the code added should be before this line.

However, more importantly adding keybinds in `common` doesn't work. `keybinding` depends on `common`, meaning the code below is called when `CBA_fnc_addKeybind` is not defined.

This means that this code needs to be moved to another component. I'm not sure if `events` is the right place or if we should add another component, just for this.